v21.1/show-backup.md

```diff
@@ -301,6 +302,91 @@ Or, use the `kms` option and the same KMS URI that was used to create the backup
 (20 rows)
 ~~~
 
+### Show a backup's internal metadata
+
+Use the `WITH as_json` option to output a backup's internal metadata, contained in its manifest file, as a JSON value:
+
+{% include copy-clipboard.html %}
+~~~ sql
+SHOW BACKUP '/2021/11/15-150703.21' IN 's3://test/backup-test?AWS_ACCESS_KEY_ID=[placeholder]&AWS_SECRET_ACCESS_KEY=[placeholder]' WITH as_json;
+~~~
+
+The response will include a `manifest` column with the file's contents as the JSON value. Use [JSONB functions](functions-and-operators.html#jsonb-functions) to query particular data or edit the format of the response.
+
+{{site.data.alerts.callout_info}}
+The response returned from `SHOW BACKUP ... WITH as_json` is a backup's internal metadata; this should **only** be used for debugging purposes. The content in the response could change and as a result should **not** be used in the same way as a stable, public API.
+{{site.data.alerts.end}}
+
+For example, to return the JSON response as a [`string`](string.html) indented and with newlines use the [`jsonb_pretty()`](functions-and-operators.html#jsonb-functions) function:
+
+{% include copy-clipboard.html %}
+~~~ sql
+SELECT jsonb_pretty(f.manifest) FROM [SHOW BACKUP '/2021/11/15-150703.21' IN 's3://test/backup-test?AWS_ACCESS_KEY_ID=[placeholder]&AWS_SECRET_ACCESS_KEY=[placeholder]' WITH as_json] AS f;
+~~~
+
+~~~
+. . .
+"S3Config": {
```

Comment:
I get it and agree. My natural inclination is to lean toward showing "more" educationally in docs. _But_, in this case, if we're already providing a note saying this is subject to change, then this could potentially be misleading if not updated quickly enough after changes to the output (which might even be difficult to track). I'm going to go ahead and reduce the output here. Thank you for the feedback @dt